articles/container-registry/container-registry-tutorial-sign-build-push.md

```diff
@@ -24,20 +24,17 @@ In this tutorial:
 
 ## Prerequisites
 
-> * Install, create and sign in to [ORAS artifact enabled registry](./container-registry-oras-artifacts.md#create-oras-artifact-enabled-registry)
+> * Install, create and sign in to OCI artifact enabled registry ACR
```

Comment:
Install, create and sign in to OCI artifact enabled registry ACR => Create and sign in ACR with OCI artifact enabled